### PR TITLE
[SM-2041] Fix some checks and bugs

### PR DESCRIPTION
**Pull request by ghost**
Added 2 new commands:
* `CheckBrandStatus`: It will check the brand creation and update the history's `status` to `approved` if it passed validation from Twilio
    * https://github.com/PerfectDayLLC/TwilioA2PBundle/blob/7f3d563b4d1361c0d92d384a2ab2abc5d8de530f/src/Console/CheckBrandStatus.php#L13-L52
* `FixCustomerProfileEvaluationProcess`: It will re-send `EndUserCustomerProfileInfo` with the new `first name`, `last name`, `email` and `phone` and then trigger the job that triggers the _evaluation_ check, so it can be re-checked
    * https://github.com/PerfectDayLLC/TwilioA2PBundle/blob/7f3d563b4d1361c0d92d384a2ab2abc5d8de530f/src/Console/FixCustomerProfileEvaluationProcess.php#L17-L91

### CODE REVIEW

**Review by joshmakar — DISMISSED** [review]

**Review by danbadds38 — DISMISSED** [review]
Great Work 🥳

**Review by joshmakar — DISMISSED** [review]
Approved with suggestions.

File: src/Console/CheckBrandStatus.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Log;
+use PerfectDayLlc\TwilioA2PBundle\Contracts\ClientRegistrationHistory as ClientRegistrationHistoryContract;
+use PerfectDayLlc\TwilioA2PBundle\Facades\EntityRegistrator as EntityRegistratorFacade;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Throwable;
+
+class CheckBrandStatus extends AbstractCommand
+{
+    protected $signature = 'a2p:check-brand-status {entity?}';
```
[the remark this answers is not in the document]
Comment (mlauritiRockstarcoders) [reply src/Console/CheckBrandStatus.php:15]: Good idea

File: src/Console/FixCustomerProfileEvaluationProcess.php [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+use PerfectDayLlc\TwilioA2PBundle\Entities\Status;
+use PerfectDayLlc\TwilioA2PBundle\Facades\EntityRegistrator as EntityRegistratorFacade;
+use PerfectDayLlc\TwilioA2PBundle\Models\ClientRegistrationHistory;
+use stdClass;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Throwable;
+
+class FixCustomerProfileEvaluationProcess extends AbstractCommand
+{
+    protected $signature = 'a2p:fix-customer-profile-evaluation-process {entity?}';
```
[the remark this answers is not in the document]
Comment (mlauritiRockstarcoders) [reply src/Console/FixCustomerProfileEvaluationProcess.php:19]: Good idea

File: src/Console/RegisterClients.php [hunk 3]
```diff
@@ -2,16 +2,19 @@
...
 use PerfectDayLlc\TwilioA2PBundle\Entities\Status;
+use PerfectDayLlc\TwilioA2PBundle\Facades\EntityRegistrator as EntityRegistratorFacade;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Throwable;
 
-class RegisterClients extends Command
+class RegisterClients extends AbstractCommand
 {
-    protected $signature = 'a2p:client-register';
+    protected $signature = 'a2p:client-register {entity?}';
```
[the remark this answers is not in the document]
Comment (mlauritiRockstarcoders) [reply src/Console/RegisterClients.php]: Good idea

**Review by danbadds38 — APPROVED** [review]
🏆

**Pull request merged**